src/index.html

```diff
@@ -8,22 +8,8 @@
 
     <title page-title>Bitwarden Web Vault</title>
 
-    <script>
-        (function () {
-            // Set theme on page load
-            // This is done outside the Angular app to avoid a flash of unthemed content before it loads
-            let theme = window.localStorage.getItem('theme');
-            if (theme?.indexOf('system') > -1) {
-                theme = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
-            } else if (theme?.indexOf('dark') > -1) {
-                theme = 'dark';
-            }
-            else {
-                theme = 'light';
-            }
-            document.documentElement.classList.add('theme_' + theme);
-        })();
-    </script>
+    <script src="scripts/theme.js" crossorigin="anonymous" type="application/javascript"
+        integrity="lmiHgZz6uiAz29i3Q4cALy8IopsEBVAk/6LBcYFMHaDKwA/Cyfod8m5wuqSCjrYL"></script>
```

Comment:
The integrity attribute needs the SHA version as a prefix so it would be `integrity="sha512-lmiHgZz6..."` or whichever SHA you used to generate the hash

[Using Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity#using_subresource_integrity)

Reply:
Thanks, this should be fixed now